isb_web/main.py

```diff
@@ -440,6 +443,47 @@ async def get_thing(
     )
 
 
+@app.get(f"/{STAC_ITEM_URL_PATH}/{{identifier:path}}", response_model=typing.Any)
+async def get_stac_item(
+    identifier: str,
+    session: Session = Depends(get_session),
+):
+    # stac wants things to have filenames, so let these requests work, too.
+    if identifier.endswith(".json"):
```

Comment:
The spec calls for links to items to have the `.json` extension, so chop off the extension if its present.